test_geosource/settings.py

```diff
@@ -136,3 +137,8 @@
         'django_filters.rest_framework.DjangoFilterBackend',
     ]
 }
+
+GEOSOURCE_LAYER_CALLBACK = 'django_geosource.geostore_callbacks.layer_callback'
+GEOSOURCE_FEATURE_CALLBACK = 'django_geosource.geostore_callbacks.feature_callback'
+GEOSOURCE_CLEAN_FEATURE_CALLBACK = 'django_geosource.geostore_callbacks.clear_features'
+GEOSOURCE_DELETE_LAYER_CALLBACK = 'django_geosource.geostore_callbacks.delete_layer'
```

Comment:
Where are these used? Also `GEOSOURCE_CLEAN_FEATURE_CALLBACK` could match function name, e.g. GEOSOURCE_CLEAR_FEATURES_CALLBACK or the other way around

Reply:
I can change it but we may need to change it in all projects using the callback.

Reply:
And i don't know which one use it.

Reply:
This is a library, it should be documented :)

Reply:
I mean, GEOSOURCE_CLEAN_FEATURE_CALLBACK allow to change the clear_features method of geosource. You can choose which feature from any library you want to use.

Here this 4 settings are just used for testing the model methods clear_features, delete, update_feature, get_layer.

When i say that i can't change the name of settings, it's because every project can change the method used. If we change the name, every project won't work anymore.